site/content/docs/02-component-format.md

```diff
@@ -354,7 +353,9 @@ The `-global-` part will be removed when compiled, and the keyframe then be refe
 
 ```html
 <style>
-	@keyframes -global-my-animation-name {...}
+	@keyframes -global-my-animation-name {
+		...;
```

Comment:
OK this one seems like a prettier autoformat gone wrong. Maybe this should be a comment like `/* Keyframe code goes here */`, to avoid same issue with future contributors?

Reply:
yeah, i like that

```suggestion
		/* code goes here */
```